data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Барфилд", "Barfield" ],
+      [ "Бас", "Bass" ],
+      [ "Бассетт", "Bassett" ],
+      [ "Баумана", "Bauman" ],
+      [ "Бегай", "Begay" ],
+      [ "Бейн", "Bain" ],
+      [ "Беланжер", "Belanger" ],
+      [ "Уайт", "White" ],
+      [ "Бердик", "Burdick" ],
+      [ "Берк", "Burk" ],
+      [ "Беркетт", "Burkett" ],
+      [ "Беркс", "Burks" ],
+      [ "Беркхарт", "Burkhart" ],
+      [ "Бернар", "Bernard" ],
+      [ "Бернс", "Burns" ],
+      [ "Бернье", "Bernier" ],
+      [ "Берр", "Burr" ],
+      [ "Беррис", "Burris" ],
+      [ "Берт", "Burt" ],
+      [ "Бертон", "Burton" ],
+      [ "Биби", "Beebe" ],
+      [ "Бизли", "Beasley" ],
+      [ "Бёрч", "Birch" ],
+      [ "Бланд", "Bland" ],
+      [ "Блаунт", "Blount" ],
+      [ "Блек", "Black" ],
+      [ "Блидсо", "Bledsoe" ],
+      [ "Блэкбёрн", "Blackburn" ],
+      [ "Бин", "Bean" ],
+      [ "Баттл", "Battle" ],
+      [ "Боннэр", "Bonner" ],
+      [ "Борхес", "Borges" ],
+      [ "Ботельо", "Botelho" ],
+      [ "Боуг", "Baugh" ],
+      [ "Бошан", "Beauchamp" ],
+      [ "Брайс", "Brice" ],
+      [ "Брантли", "Brantley" ],
+      [ "Бранхам", "Branham" ],
+      [ "Брасвелл", "Braswell" ],
+      [ "Брей", "Bray" ],
+      [ "Брейди", "Brady" ],
+      [ "Бренд", "Brand" ],
+      [ "Брубейкер", "Brubaker" ],
+      [ "Бруссар", "Broussard" ],
+      [ "Брэннон", "Brannon" ],
+      [ "Бургер", "Burger" ],
+      [ "Бурлесон", "Burleson" ],
+      [ "Куик", "Quick" ],
+      [ "Бьюли", "Beaulieu" ],
+      [ "Бэбб", "Babb" ],
+      [ "Бэрри", "Berry" ],
+      [ "Бэррон", "Barron" ],
+      [ "Бёрден", "Burden" ],
+      [ "Бёрдетт", "Burdette" ],
+      [ "Бёрнем", "Burnham" ],
+      [ "Вайз", "Wise" ],
+      [ "Вайзмен", "Wiseman" ],
+      [ "Валентайн", "Valentine" ],
+      [ "Ванхорн", "Vanhorn" ],
+      [ "Ватсон", "Watson" ],
+      [ "Ватт", "Watt" ],
+      [ "Вейр", "Ware" ],
+      [ "Джолли", "Jolly" ],
+      [ "Хэм", "Ham" ],
+      [ "Виггинс", "Wiggins" ],
+      [ "Видаль", "Vidal" ],
+      [ "Виджид", "Vigil" ],
+      [ "Виейра", "Vieira" ],
+      [ "Вишневский", "Wisniewski" ],
+      [ "Дэнт", "Dent" ],
+      [ "Волк", "Wolf" ],
+      [ "Уилл", "Will" ],
+      [ "Вомак", "Womack" ],
+      [ "Гейтс", "Gates" ],
+      [ "Ист", "East" ],
+      [ "Хармс", "Harms" ],
+      [ "Вульф", "Wolfe" ],
+      [ "Вутен", "Wooten" ],
+      [ "Вьюер", "Weaver" ],
+      [ "Вэлш", "Welsh" ],
+      [ "Уайатт", "Wyatt" ],
+      [ "Габриэль", "Gabriel" ],
+      [ "Гавани", "Havens" ],
+      [ "Галант", "Gallant" ],
+      [ "Галлардо", "Gallardo" ],
+      [ "Галлоуэй", "Galloway" ],
+      [ "Гальвез", "Galvez" ],
+      [ "Ганье", "Gagne" ],
+      [ "Гарза", "Garza" ],
+      [ "Гарланд", "Garland" ],
+      [ "Гарретт", "Garrett" ],
+      [ "Гейгера", "Geiger" ],
+      [ "Гейл", "Gale" ],
+      [ "Герра", "Guerra" ],
+      [ "Гиллиленд", "Gilliland" ],
+      [ "Чейз", "Chase" ],
+      [ "Говард", "Howard" ],
```

Comment:
This one seems straightforward but isn't.

As a given name, it would most-likely be translated as `Говард`, as is the convention. As a *family* name, however... It appears that today, `Ховард` would win, given (as per my limited research)

* [Рон Ховард](https://ru.wikipedia.org/wiki/%D0%A5%D0%BE%D0%B2%D0%B0%D1%80%D0%B4,_%D0%A0%D0%BE%D0%BD)
* [Брайс Даллас Ховард](https://ru.wikipedia.org/wiki/%D0%A5%D0%BE%D0%B2%D0%B0%D1%80%D0%B4,_%D0%91%D1%80%D0%B0%D0%B9%D1%81_%D0%94%D0%B0%D0%BB%D0%BB%D0%B0%D1%81) (I suppose it helps that these two are related)
* [Рэнс Ховард](https://ru.wikipedia.org/wiki/%D0%A5%D0%BE%D0%B2%D0%B0%D1%80%D0%B4,_%D0%A0%D1%8D%D0%BD%D1%81)
* [Шанте Лоу](https://ru.wikipedia.org/wiki/%D0%9B%D0%BE%D1%83,_%D0%A8%D0%B0%D0%BD%D1%82%D0%B5) (born Шанте Ховард)
* [Малкольм Ховард](https://ru.wikipedia.org/wiki/%D0%A5%D0%BE%D0%B2%D0%B0%D1%80%D0%B4,_%D0%9C%D0%B0%D0%BB%D0%BA%D0%BE%D0%BB%D1%8C%D0%BC)

At the same time,

* [Мишель Говард](https://ru.wikipedia.org/wiki/%D0%93%D0%BE%D0%B2%D0%B0%D1%80%D0%B4,_%D0%9C%D0%B8%D1%88%D0%B5%D0%BB%D1%8C) (notable because she's alive as if this comment)
* [Люк Говард](https://ru.wikipedia.org/wiki/%D0%93%D0%BE%D0%B2%D0%B0%D1%80%D0%B4,_%D0%9B%D1%8E%D0%BA)
* [Эбенизер Говард](https://ru.wikipedia.org/wiki/%D0%93%D0%BE%D0%B2%D0%B0%D1%80%D0%B4,_%D0%AD%D0%B1%D0%B5%D0%BD%D0%B8%D0%B7%D0%B5%D1%80)

`Говард` seems to be considered an outdated form, as far as surnames go. I believe that it's fine as it is for the game, because the same name could represent both a given name and a family name, and the way I see it, `Говард` would be expected as the first name, with no particular preference given to the spelling of it as a last name. Still, I figured it was worth noting that there's a bit of a distinction here.